lnbits/wallets/__init__.py

```diff
@@ -16,6 +16,7 @@
 from .corelightningrest import CoreLightningRestWallet
 from .eclair import EclairWallet
 from .fake import FakeWallet
+from .alby import AlbyWallet
```

Comment:
This list hast to be sorted, otherwise `ruff` complains.
Move alby just before the cliche.